From 69f6df9961922da068226200cae722c34ca21866 Mon Sep 17 00:00:00 2001
From: Vicentiu Galanopulo <vicentiu@balena.io>
Date: Mon, 16 Nov 2020 17:01:37 +0100
Subject: [PATCH] Set GPIO5_IO22 to low

GPIO5_IO22 will be used to wake the board from deep sleep.
It will have to be set to LOW in board_init as this is the
required start state of the deep sleep implementation

Upstream-Status: Inappropriate [configuration]
Signed-off-by: Vicentiu Galanopulo  <vicentiu@balena.io>
---
 board/compulab/cl-som-imx8/cl-som-imx8.c | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/board/compulab/cl-som-imx8/cl-som-imx8.c b/board/compulab/cl-som-imx8/cl-som-imx8.c
index 2d881d0..bdac8c7 100644
--- a/board/compulab/cl-som-imx8/cl-som-imx8.c
+++ b/board/compulab/cl-som-imx8/cl-som-imx8.c
@@ -344,6 +344,9 @@ int board_init(void)
 	gpio_request(IMX_GPIO_NR(4, 7),"uPCIe_1_RST");
 	gpio_direction_output(IMX_GPIO_NR(4, 7), 1);
 
+	gpio_request(IMX_GPIO_NR(5, 22),"pm_uart1_rx");
+	gpio_direction_output(IMX_GPIO_NR(5, 22), 0);
+
 #if defined(CONFIG_USB_DWC3) || defined(CONFIG_USB_XHCI_IMX8M)
 	init_usb_clk();
 #endif
-- 
2.7.4

